events/templates/events/events.html

```diff
@@ -45,6 +45,27 @@
 
 <div class="sticky-note-container-big mt-5">
     <h1>Events</h1>
+
+    <!-- Filtering form -->
+    <form method="get" action="{% url 'events:index' %}" class="filter-form">
+        <label for="start_time">Start Time:</label>
```

Comment:
I feel we should add checks that start time cannot be less than current time and end time cannot be less than start time

Reply:
yup, I added the code for it.

Reply:
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/127150757/323e3390-2906-4eb4-8478-068575d4effc)
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/127150757/75e69d7f-29f8-4f6f-8562-7d4aaa6eb6de)